src/main/java/com/carhouse/controller/CarModelRestController.java

```diff
@@ -10,12 +10,12 @@
 import java.util.List;
 
 /**
- * Ajax Rest Controller is used to perform queries to server with ajax without updating page.
+ * CarModelRestController is used to get car model list with ajax.
```

Comment:
that's why I usually prefer to write `Class responsible for` or `Class represents`